en/advanced_config/parameter_reference.md

```diff
@@ -13101,7 +13101,7 @@ table {
 </tr>
 <tr>
  <td><strong id="GPS_YAW_OFFSET">GPS_YAW_OFFSET</strong> (FLOAT)</td>
- <td>Heading/Yaw offset for dual antenna GPS <p><strong>Comment:</strong> Heading offset angle for dual antenna GPS setups that support heading estimation. Set this to 0 if the antennas are parallel to the forward-facing direction of the vehicle and the first antenna is in front. The offset angle increases clockwise. Set this to 90 if the first antenna is placed on the right side and the second on the left side of the vehicle.</p>   <p><b>Reboot required:</b> true</p>
+ <td>Heading/Yaw offset for dual antenna GPS <p><strong>Comment:</strong> Heading offset angle for dual antenna GPS setups that support heading estimation. Set this to 0 if the antennas are parallel to the forward-facing direction of the vehicle and the Rover antenna is in front. The offset angle increases clockwise. Set this to 90 if the Rover antenna is placed on the right side and the Moving Base on the left side of the vehicle.</p>   <p><b>Reboot required:</b> true</p>
```

Comment:
@samchamberlin The parameter reference is dynamically created from the autopilot source. I have created a PR to fix it there (note, used lower case rather than capitalised Rover and moving Base: https://github.com/PX4/PX4-Autopilot/pull/19113

Can you undo this file change in this PR?

Reply:
Done.